pass-nihms-loader/nihms-pass-client/src/test/java/org/eclipse/pass/client/nihms/NihmsPassClientServiceIT.java

```diff
@@ -63,28 +63,138 @@ public void lookupJournalByIssn() throws IOException {
         passClient.createObject(journal);
         String journalId = journal.getId();
 
-        String journalFilterFoo = RSQL.equals(ISSNS_FLD, "fooissn");
-        PassClientSelector<Journal> journalFooSelector = new PassClientSelector<>(Journal.class);
-        journalFooSelector.setFilter(journalFilterFoo);
-        PassClientResult<Journal> journalFooResult = passClient.selectObjects(journalFooSelector);
-        List<Journal> journalsFoo = journalFooResult.getObjects();
-
-        String journalFilterBar = RSQL.equals(ISSNS_FLD, "barissn");
-        PassClientSelector<Journal> journalBarSelector = new PassClientSelector<>(Journal.class);
-        journalFooSelector.setFilter(journalFilterBar);
-        PassClientResult<Journal> journalBarResult = passClient.selectObjects(journalBarSelector);
-        List<Journal> journalsBar = journalBarResult.getObjects();
-
-        assertEquals(1, journalsFoo.size());
-        assertEquals(journalId, journalsFoo.get(0).getId());
-
-        assertEquals(1, journalsBar.size());
-        assertEquals(journalId, journalsBar.get(0).getId());
-
         assertEquals(journalId, underTest.findJournalByIssn("fooissn"));
         assertEquals(journalId, underTest.findJournalByIssn("barissn"));
 
         // and that a lookup by a non-existent issn returns null.
         assertNull(underTest.findJournalByIssn("nonexistentissn"));
     }
+
+    /**
+     * Using different variants of the same award number, demonstrate that normalized award numbers are found using
+     * different variants of an award number e.g. R01AR074846 should match R01AR074846-01A1 or 1R01AR074846-01
+     * the activity code, institute code and serial number are the minimum set of strings required to match
+     */
+    @Test
+    public void shouldFindNihGrantAwardNumber() throws IOException {
+        Grant grant1 = new Grant("1");
```

Comment:
Interesting that the id is set here. It's not needed and if elide actually creates the objects with that id you might run into collisions with other tests.

Reply:
Done.